phys2bids/phys2bids.py

```diff
@@ -212,8 +212,9 @@ def _main(argv=None):
     print('Reading infos')
     phys_in.print_info(options.filename)
     # #!# Here the function viz.plot_channel should be called
-    # for the desired channels.
-
+    if options.chplot != '' or options.info:
```

Comment:
If `option.info`, but `options.chplot == ''`, there won't be a path to save the plot.
```suggestion
    if options.chplot != '' or options.info:
        if options.chplot == '':
            options.chplot == f'{infile}.png'
```

Reply:
This is already managed inside the function don't worry about it